### PR TITLE
Add login modal

### PR DESCRIPTION
**Pull request by richtier**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if updating requirements) Requirements have been compiled.

The modal is not really exciting. The key thing is the build tool works :)

- outputs individual component files
- outputs bundle of all component files

login modal gif

### CODE REVIEW

File: config/urls.py [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 
+import sso.urls
+
 
 urlpatterns = [
-    path(r'django-admin/', admin.site.urls),
-    path(r'admin/', include(wagtailadmin_urls)),
-    path(r'documents/', include(wagtaildocs_urls)),
 
+    url(r'^django-admin/', admin.site.urls),
```
Comment (sdonk) [inline config/urls.py]: Can you please use `path`? That's the recommended function in Django 2

File: sso/urls.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import url
+
+import sso.views
+
+app_name = 'sso'
+
+
+urlpatterns = [
+    url(
```
Comment (sdonk) [inline sso/urls.py]: path

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,3 +5,4 @@
 ### Fixed bugs
 
 ### Implemented enhancements
+- MVP-79 - Add react built tool
```
Comment (eadpearce) [inline CHANGELOG.md]: *build ;)

File: javascript/src/LoginModal.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
+import Modal from 'react-modal';
+
+
+export const MESSAGE_INCORRECT_CREDENTIALS = 'Incorrect username or password';
+export const MESSAGE_UNEXPEXCTED_ERROR = 'Unexpected Error';
```
Comment (eadpearce) [inline javascript/src/LoginModal.js]: *UNEXPECTED ?

**Pull request merged**